.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.7', '3.8', '3.9', '3.10']
+        python-version: ['3.8', '3.9', '3.10', '3.11']
```

Comment:
https://github.com/fidelity/textwiser?tab=readme-ov-file#installation
We stil say 3.6+, which seems inaccurate. Given that these versions of Python are decommisioned now, shall we jump to Python 3.8+? This needs changing the README and possible the Docs?

Reply:
@kuppulur shall we update the docs as well? see the link above

Reply:
I updated the docs as well. Thanks for the comment.